src/main/java/org/broadinstitute/hellbender/cmdline/CommandLineProgram.java

```diff
@@ -161,6 +169,8 @@ public Object instanceMainPostParseArgs() {
                 );
                 final boolean usingIntelDeflater = (BlockCompressedOutputStream.getDefaultDeflaterFactory() instanceof IntelDeflaterFactory && ((IntelDeflaterFactory)BlockCompressedOutputStream.getDefaultDeflaterFactory()).usingIntelDeflater());
                 logger.info("Deflater " + (usingIntelDeflater ? "IntelDeflater": "JdkDeflater"));
+                final boolean usingIntelInflater = (BlockGunzipper.getDefaultInflaterFactory() instanceof IntelInflaterFactory && ((IntelInflaterFactory)BlockGunzipper.getDefaultInflaterFactory()).usingIntelInflater());
+                logger.info("Inflater " + (usingIntelInflater ? "IntelInflater": "JdkInflater"));
```

Comment:
What happens if the user requests the intel inflater and it fails to load? Is there a user-friendly log message in that case?